plugins/star-rating/frontend/public/templates/star.html

```diff
@@ -328,24 +328,24 @@
                         <div class="emotions-area">
                             <p id="question-area">What is your opinion <br>of this page?</p>
                             <div class="emotion-first tooltip-wrapper">
-                                <span class="tooltip-text">Terrible</span>
-                                <img class="grow rating-emotion tooltip" title="Terrible" data-score="1" src="star-rating/images/star-rating/0_gray.svg">
+                                <span style="left: -8px !important;" class="tooltip-text">Terrible</span>
+                                <img class="grow rating-emotion tooltip" data-score="1" src="star-rating/images/star-rating/0_gray.svg">
                             </div>
                             <div class="emotion tooltip-wrapper">
-                                <span class="tooltip-text">Bad</span>
-                                <img title="Bad" class="grow rating-emotion tooltip" data-score="2" src="star-rating/images/star-rating/1_gray.svg">
+                                <span style="left: -4px !important;" class="tooltip-text">Bad</span>
+                                <img class="grow rating-emotion tooltip" data-score="2" src="star-rating/images/star-rating/1_gray.svg">
                             </div>
                             <div class="emotion tooltip-wrapper">
-                                <span class="tooltip-text">Okay</span>
-                                <img title="Okay" class="grow rating-emotion tooltip" data-score="3" src="star-rating/images/star-rating/2_gray.svg">
+                                <span style="left: -4px !important;" class="tooltip-text">Okay</span>
+                                <img class="grow rating-emotion tooltip" data-score="3" src="star-rating/images/star-rating/2_gray.svg">
                             </div>
                             <div class="emotion tooltip-wrapper">
-                                <span class="tooltip-text">Nice</span>
-                                <img title="Nice" class="grow rating-emotion tooltip" data-score="4" src="star-rating/images/star-rating/3_gray.svg">
+                                <span style="left: -4px !important;" class="tooltip-text">Nice</span>
+                                <img class="grow rating-emotion tooltip" data-score="4" src="star-rating/images/star-rating/3_gray.svg">
                             </div>
                             <div class="emotion tooltip-wrapper">
-                                <span class="tooltip-text">Great</span>
-                                <img title="Great" class="grow rating-emotion tooltip" data-score="5" src="star-rating/images/star-rating/4_gray.svg">
+                                <span style="left: -4px !important;" class="tooltip-text">Great</span>
```

Comment:
If you use !important here, it becomes very difficult for themes to override the look and feel of the UI. Do you need that though? Doesn't an inline style always override any pre-existing CSS?

Reply:
Ideally there should be no inline css at all, and all should go to CSS files and without important :)